js/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -0,0 +1,92 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * @param {object} app The app
+ * @param {object} args An arguments object with usedKeywords, searchUrl, postUrl,
```

Comment:
Please add separate param options for the possible properties within the args object.

See scoreformatter for an example of this.